AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+                f"P0: Light component failed to set {light_type} type")
+            general.idle_wait_frames(1)
             Report.result(test_light_type, current_light_type == LIGHT_TYPES[light_type])
 
-        # 6. Enter/Exit game mode.
-        TestHelper.enter_game_mode(Tests.enter_game_mode)
-        general.idle_wait_frames(1)
-        TestHelper.exit_game_mode(Tests.exit_game_mode)
+            # Exit loop for unknown light type.
+            if current_light_type == 0:
+                continue
 
-        # 7. Test IsHidden.
-        light_entity.set_visibility_state(False)
-        Report.result(Tests.is_hidden, light_entity.is_hidden() is True)
+            # 7. Check for Shape component.
+            if current_light_type in (1, 2, 3, 4, 5):
+                light_shape = LIGHT_SHAPES[light_type]
+                test_light_shape = (
+                    f"{light_shape} present",
+                    f"P1: {light_shape} was not found")
+                Report.result(test_light_shape,
+                              light_entity.has_component(light_shape))
 
-        # 8. Test IsVisible.
-        light_entity.set_visibility_state(True)
-        general.idle_wait_frames(1)
-        Report.result(Tests.is_visible, light_entity.is_visible() is True)
+            # 8. Edit Color parameter.
+            color_value = math.Color(1.0, 0.0, 0.0, 1.0)
+            light_component.set_component_property_value(AtomComponentProperties.light('Color'), color_value)
+            light_color = light_component.get_component_property_value(
+               AtomComponentProperties.light('Color'))
+            general.idle_wait_frames(1)
+            Report.result(Tests.edit_light_color, light_color.IsClose(color_value))
+
+            # 9. Set Intensity mode.
+            # if current_light_type < 6:
+            #     for intensity_mode in INTENSITY_MODE.keys():
+            #         light_component.set_component_property_value(
+            #             AtomComponentProperties.light('Intensity mode'), INTENSITY_MODE[intensity_mode])
+            #         current_intensity_mode = light_component.get_component_property_value(
+            #             AtomComponentProperties.light('Intensity mode'))
+            #         test_intensity_mode = (
+            #             f"Intensity mode set to {current_intensity_mode}",
+            #             f"P1: Intensity mode failed to be set to {current_intensity_mode}")
+            #         Report.result(test_intensity_mode,
+            #                       current_intensity_mode == INTENSITY_MODE[intensity_mode])
+
+            # 10. Edit the Intensity parameter.
+            light_component.set_component_property_value(AtomComponentProperties.light('Intensity'), 1000)
+            Report.result(Tests.edit_intensity_value,
+                          light_component.get_component_property_value(
+                              AtomComponentProperties.light('Intensity')) == 1000)
```

Comment:
is intensity an integer or decimal. if it isn't integer you might need to sue IsClose for float variance

Reply:
It is a float, which does not have an IsClose attribute.